test/Microsoft.AspNetCore.Mvc.Razor.Test/Internal/FileProviderRazorProjectTest.cs

```diff
@@ -18,7 +19,9 @@ public void EnumerateFiles_ReturnsEmptySequenceIfNoCshtmlFilesArePresent()
             var file2 = fileProvider.AddFile("File2.js", "content");
             fileProvider.AddDirectoryContent("/", new IFileInfo[] { file1, file2 });
 
-            var razorProject = new FileProviderRazorProject(fileProvider);
+            var accessor = Mock.Of<IRazorViewEngineFileProviderAccessor>(a => a.FileProvider == fileProvider);
```

Comment:
I've actually become a huge fan of this, it scales way better than their builder API